src/login/Login.tsx

```diff
@@ -68,6 +68,9 @@ const Login = () => {
               name="username"
               value={username}
               onChange={onUsernameChange}
+              isRequired
+              isInvalid={!!loginError && !username}
+              feedback={!!loginError && !username ? 'Username is required.' : undefined}
```

Comment:
I think that we should move the "login error" message, to be an object that looks like: 

{
	message: 'some error message',
	username: 'some login error message specific to username'
	password: 'some login error messages specific to password'
}

This is consistent with the rest of the application, and I think simplifies the ternaries since
`isInvalid = error?.username` and feedback is `t(error?.username)`